server/plugins/task-plugin-postgres/pom.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
have we tested import/export for postgresql?

we'd better remove this empty task plugin if we do't support it actually~

Reply:
this module has been removed